### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by soesandarwin2201**
## What does this PR do?
This PR is raised to make the capstone-project for udemy home page and about page .
## Description of the task to be completed
- [x] Make the respo setup
- [x] make the home page index file and css file for desktop
- [x] make the home page for mobile version
- [x] make the mobile nav menu work with javascript
- [x] create new branch for about and make mistaked with the branch
- [x] make feature section with the stored data of javascript
- [x] make the about page index.html and style for desktop version
- [x] make the style for mobile version
- [x] reslove the conflict between branches
- [x] check the links work
- [x] check the linters error         
## How should it be tested
Clone the repository
```
git clone https://github.com/soesandarwin2201/HTML-CSS-JavaScript-capstone-project1
```
Navigate to the folder
```
cd HTML-CSS-JavaScript-capstone-project1
```
Checkout the branch
```
git checkout home-page'

### My document file link

https://www.loom.com/share/ae76a7aa331749a18a74d69ea7250161


```
#### Thanks for reviewing my PR :✨😊:

### CODE REVIEW

**Review by Bondok6 — COMMENTED** [review]
## Hi @soesandarwin2201 🙋‍♂️, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project, that doesn't have JavaScript Linters (Eslint), kindly add it.
For this project you should use [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) Linters.
1. Linter should be set up correctly for this repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,27 @@
...
+ -Twitter: @SoeSand14306315
+ 
+ ## Contributing
+ - Contributions, issues, and feature requests are welcome!
+
+   Feel free to check the issues page.
+   
+ #Show your support
+ 
+ Give a ⭐️ if you like this project!
```
Comment (OLIPLICHE) [inline README.md]: - [x] Please make sure that this Readme contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: README.md [hunk 2]
```diff
@@ -1 +1,27 @@
...
\ No newline at end of file
+# HTML-CSS-JavaScript-capstone-project1
+# capstone-project for udemy web course
+# mobile & desktop version
+# home page & about page
+
+## Build with
+ - Html
+ - Css
+ - javascript
```
Comment (OLIPLICHE) [inline README.md]: # Optional:
- Please make your Readme file more descriptive and Professional. Open this link 👉 [Readme Template ](https://github.com/microverseinc/readme-template/blob/master/README.md)and follow this example.

File: README.md [hunk 3]
```diff
@@ -1 +1,27 @@
...
+ - flexbox
+ - grid
+
```
Comment (OLIPLICHE) [inline README.md]: - [ ] **Live version and project requirements**: Please make use of GitHub Pages to deploy the web pages.
- The project must be deployed properly: all pages, links, and images work as expected.
Please follow the  👉 [ project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+     <li><a href="#about.html">Sponsor</a></li>
+     <li><a href="#about.html">News</a></li>
+     <li><a href="#about.html" class="business">Udemy Business</a></li>
+     
+    </ul>
+  </nav>
+
+<nav id="toggle" class="mobile">
+    <i class="uil uil-bars mobile-menu"></i>
+  </nav>
```
Comment (OLIPLICHE) [inline index.html]: # Mobile Size:
- Its background is transparent or the same color/pattern as the top section background.
- See your design: 
Capture
- Original Design:
- 
Capture

File: about.html [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+  <!--end of about home section-->
+
+  <!--about transform life section-->
+  <section class="life">
+   <h2 class="life-title">Transform your life through education</h2>
+   <p class="life-text desktop">Learners around the world are launching new careers, advancing in their fields, and enriching their lives.Who doesn’t love a good story of self-reinvention We’ve certainly seen our share among Udemy students and instructors alike There’s the Brazilian lawyer who became an app developer, the Egyptian civil engineer who pivoted to sports management, the New York actor who turned into a startup CEO, and many more...</p>
+
+   <p class="life-text mobile">Learners around the world are launching new careers, advancing in their fields, and enriching their lives</p>
+
+   <img src="./images/Udemy_logo.svg (1).png" alt="udemy logo images" class="udemy-logo">
```
Comment (OLIPLICHE) [inline about.html:116]: - [x] About page: 
- [x] Please the middle section is overlaping. Kindly fix it. Also, make the section looks nice
 
Capture

File: about.html [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  </section>
+
+  <footer class="footer about-footer">
+    <a href="index.html"><img src="./images/logo1.png" alt="Udemy logo" class="logo footer-logo"></a>
+
+    <div class="footer-text">
+      <p class="footer-copy">2022 All Rights served to Soe Sandar Win</p>
+      <p class="footer-copy">At 65+ hours, this Web Development course is without a doubt the most comprehensive web development course available online.</p>
+    </div>
+  </footer>
```
Comment (OLIPLICHE) [inline about.html:174]: # Mobile size "About": footer color is white
Please make sure to follow the design. 
- From your page 
- 
Capture
- From the original design:
- 
Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @soesandarwin2201, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️

- [x]  Please provide a recorded video presenting the features of the project. The maximum length of the video must be 5 minutes. Also, the video must contain a highlight about a piece of the code you find interesting or proud of making. It is all part of your project requirements.

_**Check the comments under the review**._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @soesandarwin2201, 

Your capstone project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

Cheers, good luck in the next modules and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**